### PR TITLE
Allow toggling the visibility of GUIs by blinking

### PR DESCRIPTION
**Pull request by S-Dafarra**
I reused some code I had for the gaze control. Hence, the modifications are a bit consistent

### CODE REVIEW

**Comment by S-Dafarra** [comment]
The PR is ready to be reviewed, I have tested it with the robot

File: modules/SRanipal_module/include/AdvancedJoypad.hpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+#include <yarp/os/ResourceFinder.h>
+
+#include <vector>
+#include <memory>
+
+class AdvancedJoypad
+{
+
+    std::shared_ptr<VRInterface> m_VRInterface;
```
Comment (GiulioRomualdi) [inline modules/SRanipal_module/include/AdvancedJoypad.hpp:22]: I was wondering why this is stored in a `shared_ptr`

File: modules/SRanipal_module/src/SRanipalModule.cpp [hunk 2]
```diff
@@ -44,24 +45,19 @@ bool SRanipalModule::configure(yarp::os::ResourceFinder &rf)
...
-        } else
+        if (!m_VRInterface->configure(rf))
         {
-            yInfo() << "[SRanipalModule::configure] Skipping gaze control.";
+            yError() << "[SRanipalModule::configure] Failed to configure VR interface.";
+            return false;
         }
+    }
 
+    if (m_useEyebrows || m_useEyelids || m_useGaze || m_useAdvancedJoypad)
```
Comment (GiulioRomualdi) [inline modules/SRanipal_module/src/SRanipalModule.cpp:59]: I would avoid nesting the `if`  (this should improve the readability) 
```cpp
    if (!m_useEyebrows && !m_useEyelids && !m_useGaze && !m_useAdvancedJoypad)
```

**Review by GiulioRomualdi — COMMENTED** [review]
Just a small minor comment, feel free to merge it anyhow

File: modules/SRanipal_module/src/SRanipalModule.cpp (hunk 2 above)
Comment (S-Dafarra) [reply modules/SRanipal_module/src/SRanipalModule.cpp:59]: Not sure I understood. The version you are proposing has the opposite result. Right now I want to get into that if if any of those components is active

**Comment by S-Dafarra** [comment]
@GiulioRomualdi if you agree I would merge

**Comment by GiulioRomualdi** [comment]
Yes merge it

**Pull request merged**